### ISSUE TITLE
Timestamp 0.0 on Linux

### ISSUE THREAD

**Issue by caspervdw** [soft-matter/pims_nd2#8]
This issue was raised by @stevendbrown . (see #3)
`dTimeStart` gives 0.0 on the Linux implementation.

**Comment by stevendbrown** [comment]
This value returns the correct Julian date using pims_nd2 on OS X but remains broken in the "fixed" release for Linux available on nd2sdk.com. I've contacted LIM about the Linux SDK but have not heard back from them yet.

**Comment by stevendbrown** [comment]
The LIM team has updated the SDK available on nd2sdk.com with a new Linux SDK that has the dTimeStamp bug corrected in my hands. The MD5 sum of the corrected package is 5c0f81233fc26fa98853cad7c447a5f3.

**Comment by caspervdw** [comment]
Thanks for getting this to work! I pushed the fix for Linux, tests pass.

Could you give me a hand with the OSX implementation? I'm guessing that I need to use:

```
nd2 = cdll.LoadLibrary(os.path.join(os.path.dirname(__file__), 'ND2SDK',
                                    'osx', 'nd2sdk.framework', 'Versions', '1', 'nd2sdk'))
```

But I never worked with OSX so I'm not sure.
Also, do you think there are any obsolete files in the current osx sdk folder?

**Comment by caspervdw** [comment]
Closed as of #10
